lib/stub_ui/lib/src/ui/text.dart

```diff
@@ -4,6 +4,11 @@
 
 part of ui;
 
+// If we actually run on big endian machines, we'll need to do something smarter
+// here. We don't use [Endian.Host] because it's not a compile-time
+// constant and can't propagate into the set/get calls.
+const Endian _kFakeHostEndian = Endian.little;
```

Comment:
This constant was moved to lib/ui/natives.dart.

Does stub_ui have an equivalent of that file?  Or should the code in text.dart that uses `_kFakeHostEndian` be removed from stub_ui?

Reply:
Moved it to natives - we can leave the implementation for now, just needs to compile